lib/desired-caps.js

```diff
@@ -21,6 +21,10 @@ const desiredCapConstraints = {
       'YouiEngine',
     ]
   },
+  deviceName: {
+    presence: true,
```

Comment:
There was a unit test for this but the default capabilities allow it to be non-existent